test/distrib/cpp/run_distrib_test_cmake_arm_cross.sh

```diff
@@ -20,6 +20,9 @@ cd "$(dirname "$0")/../../.."
 # Install openssl (to use instead of boringssl)
 apt-get update && apt-get install -y libssl-dev
 
+# Install arm cross-compiler
+apt-get update && apt-get install -y g++-6-arm-linux-gnueabihf
```

Comment:
Ideally we should only be installing as part of the Dockerfile (pre-installing the compiler in the base docker image is easy).
The `apt-get install -y libssl-dev`  above is an exception since it's part of the distribtest (we "install" it as an alternative to using boringssl).

Reply:
Yeah, that's a good idea. I'm creating a new one.